src/Frontend/Path.hs

```diff
@@ -138,8 +137,8 @@ data Space =
 
 instance SOP.Generic Space
 
-viewIdea :: Idea -> Main
-viewIdea idea = IdeaPath (idea ^. ideaLocation) $ ViewIdea (idea ^. _Id)
+viewIdea :: Idea -> Maybe (AUID Comment) -> Main
```

Comment:
I would have made two functions instead of the `Maybe` (the `Maybe` in `ViewIdea` is fine though)